pkg/test/env/environment.go

```diff
@@ -212,7 +233,7 @@ func (e *Environment) LoadFixtureBuilderFactories(factories ...fixtures.FixtureB
 			return fmt.Errorf("can not build fixture builder: %w", err)
 		}
 
-		if err = e.fixtureLoader.Load(e.ctx, fixtureBuilder.Fixtures()); err != nil {
+		if err := e.fixtureLoader.Load(e.ctx, fixtureBuilder.Fixtures()); err != nil {
```

Comment:
why re-create this var? in line 229 you already have an err var defined.

Reply:
sometimes the linter complains about these things due to: https://go-critic.com/overview.html#sloppyreassign  
you get something like:
```
sloppyReassign: re-assignment to `err` can be replaced with `err := env.runner.RunContainers(skeletons)` (gocritic)
```

Reply:
strange, my goland often complains for these when i re-assign a var that exists already. But fine with me.